webpack.config.js

```diff
@@ -36,27 +36,27 @@ module.exports = {
       {
         test: /\.tsx?$/,
         use: 'ts-loader',
-        exclude: /node_modules/,
+        exclude: /node_modules/
```

Comment:
@bcoe I expected the post-processer to fix this.  Did the post-processor and linter drift apart?